src/index.ts

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-// ** This file is automatically generated by gapic-generator-typescript. **
-// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** This file is automatically generated by synthtool. **
```

Comment:
I cross referenced this with `googleapis/googleapis` and this is the up-to-date comment string, not sure why we didn't get a synthtool PR for it.